src/main/java/cinema/model/User.java

```diff
@@ -15,6 +18,8 @@
     @Column(unique = true)
     private String email;
     private String password;
+    @OneToMany(cascade = CascadeType.PERSIST)
```

Comment:
does it work with `save()` method that you use for saving User entity to DB?

I would suggest to remove this cascade and also change the relationship
as Roman asked earlier, can each role belong to one user only?

Reply:
done